ui/src/pages/notifications/NotificationComponents.tsx

```diff
@@ -174,6 +174,7 @@ export function RequestNotification<T extends Fields>({
               className="ghost"
               content="Approve"
               type="submit"
+              disabled={!lookupFields}
```

Comment:
There should also be a warning here, when fields are missing that block the user from accepting a request